### PR TITLE
EDU-961/962: history shards

### PR DESCRIPTION
**Pull request by DJSanti**
## What does this PR do?
This PR corrects some History Shard and History Service information.

## Notes to reviewers
Finally seemed to fix this branch.

### CODE REVIEW

File: docs-src/concepts/what-is-a-history-service.md [hunk 1]
```diff
@@ -7,11 +7,13 @@ tags:
...
 
-The History Service is responsible for persisting Workflow Execution state and determining what to do next to progress the Workflow Execution by using [History Shards](/concepts/what-is-a-history-shard).
+The History Service is responsible for persisting Workflow Execution state to the Workflow History.
+When the Workflow Execution is able to progress, the History Service adds a Task with the Workflow's updated history to the Task Queue.
+From there, a Worker can poll for work, receive this updated history, and resume execution.
 
 ![History Service](/diagrams/temporal-history-service.svg)
 
-The total number of History Services can be between 1 and the total number of History Shards.
+The total number of History Services can be between 1 and the total number of [History Shards](/concepts/what-is-a-history-shard).
```
Comment (robholland) [inline docs-src/concepts/what-is-a-history-service.md:16]: I would use the term "History Service process". The number of processes can be larger than the number of shards, it's just that those beyond the shard count would not have any useful work to do. It's also highly unlikely to need a History Service process per shard. As an example, we normally recommend one History Service process per 500 shards.

File: docs-src/concepts/what-is-a-history-service.md [hunk 2]
```diff
@@ -7,11 +7,13 @@ tags:
   - term
 ---
 
-The History Service is responsible for persisting Workflow Execution state and determining what to do next to progress the Workflow Execution by using [History Shards](/concepts/what-is-a-history-shard).
+The History Service is responsible for persisting Workflow Execution state to the Workflow History.
+When the Workflow Execution is able to progress, the History Service adds a Task with the Workflow's updated history to the Task Queue.
+From there, a Worker can poll for work, receive this updated history, and resume execution.
```
Comment (robholland) [inline docs-src/concepts/what-is-a-history-service.md:12]: Looks good to me 👍

File: docs-src/concepts/what-is-a-history-shard.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ tags:
   - term
 ---
 
-A History Shard is an important unit within a Temporal Cluster by which the scale of concurrent Workflow Execution throughput can be measured.
+A History Shard is an important unit within a Temporal Cluster by which concurrent Workflow Execution throughput can be scaled.
```
Comment (robholland) [inline docs-src/concepts/what-is-a-history-shard.md:10]: 👍

**Review by robholland — APPROVED** [review]
Nice :+1:

File: docs-src/concepts/what-is-a-history-service.md [hunk 4]
```diff
@@ -7,15 +7,17 @@ tags:
...
 
 ![History Service](/diagrams/temporal-history-service.svg)
```
Comment (djmagee) [inline docs-src/concepts/what-is-a-history-service.md:14]:
```suggestion
![Block diagram of how the History Service relates to the other services of the Temporal Server and to a Temporal Cluster](/diagrams/temporal-history-service.svg)
```

**Review by djmagee — APPROVED** [review]
Just one suggestion that expands the alt text for a diagram.

**Pull request merged**